app/src/client/resources.ts

```diff
@@ -116,6 +124,8 @@ export class CollectionToken {
       String(obj.name),
       obj.imageUrl ? String(obj.imageUrl) : null,
       obj.description ? String(obj.description) : null,
+      obj.attributes ? obj.attributes as TokenAttribute[] : [],
```

Comment:
`obj.attributes as TokenAttribute[]` this isnt right. look how other objects are done in lists

Reply:
a good example is the tokenTransfers in the uidata